chainforge/react-server/src/store.js

```diff
@@ -138,6 +208,11 @@ const useStore = create((set, get) => ({
     set({ AvailableLLMs: llmProviderList });
   },
 
+  aiFeaturesProvider: undefined,
+  setAIFeaturesProvider: (llmProvider) => {
+    set({ aiFeaturesProvider: llmProvider });
+  },
```

Comment:
Note that this won't be saved to localStorage as a setting, meaning users will have to set it every time. I will fix this in the near future so there's a standard place for all global settings that is also cached in the user's browser.